### ISSUE TITLE
SyntaxError: Unexpected token export

### ISSUE THREAD

**Issue by adiusz** [gperl27/gatsby-styled-components-dark-mode#23]
Hi.
I set up a new project and I have a problem in getting this plugin running. I'm following the instruction, I installed plugin, I created file theme.js in /src directory (and copied its content from instruction) and I added plugin to gatsby-config. After running `gatsby develop` I got this: 

```
We encountered an error while trying to load your site's gatsby-config. Please fix the error and try again.
  Error: /path/to/theproject/src/theme.js:1
  (function (exports, require, module, __filename, __dirname) { export c  onst darkTheme = {
                                                                ^^^^^^
  SyntaxError: Unexpected token export
```

My gatsby-config:
```
module.exports = {
  siteMetadata: {
    title: `Gatsby Default Starter`,
    description: `Kick off your next, great Gatsby project with this default starter. This barebones starter ships with the main Gatsby configuration files you might need.`,
    author: `@gatsbyjs`,
  },
  plugins: [
    `gatsby-plugin-react-helmet`,
    {
      resolve: `gatsby-source-filesystem`,
      options: {
        name: `images`,
        path: `${__dirname}/static`,
      },
    },
    `gatsby-transformer-sharp`,
    `gatsby-plugin-sharp`,
    {
      resolve: `gatsby-plugin-manifest`,
      options: {
        name: `gatsby-starter-default`,
        short_name: `starter`,
        start_url: `/`,
        background_color: `#663399`,
        theme_color: `#663399`,
        display: `minimal-ui`, // icon: `src/images/gatsby-icon.png`, // This path is relative to the root of the site.
      },
    }, // this (optional) plugin enables Progressive Web App + Offline functionality
    // To learn more, visit: https://gatsby.dev/offline
    // `gatsby-plugin-offline`,

    `gatsby-plugin-transition-link`,
    {
      resolve: `gatsby-plugin-styled-components`,
      options: {
        displayName: true
      },
    },
    {
      resolve: `gatsby-styled-components-dark-mode`,
      options: {
        light: require(`${__dirname}/src/lightTheme.js`),
        dark: require(`${__dirname}/src/darkTheme.js`),
      },
    },
    {
      resolve: "gatsby-plugin-react-svg",
      options: {
        rule: {
          include: /assets/ // See below to configure properly
        },
      }
    },

  ],
}
```

Interesting thing is that when I delete `export default lightTheme/darkTheme` from my theme files, I can get site running, but toggling between themes doesn't cause anything.

Hope some can help. Peace.

**Comment by gperl27** [comment]
It looks like you might have a syntax error in your theme files; definitely double-check those, especially since your project works without them. Feel free to put the code or a link to those files here.

Additionally for theme toggling and the like, check out the [readme's example](https://github.com/gperl27/gatsby-styled-components-dark-mode#full-example) for a simple starting implementation.

**Comment by adiusz** [comment]
Thanks for quick respond.
I tried this example and I copied all the fields to my project from it. No typing myself.
Here it is:
![Zrzut ekranu 2020-08-6 o 16 51 43](https://user-images.githubusercontent.com/20825919/89546657-1d865380-d805-11ea-97ec-693ac7b3bed2.png)

And here is the content of them:

```
const darkTheme = {
  mainColor: `#010101`,
  secondaryColor: `#052192`,
}
```

```
const lightTheme = {
  mainColor: 'pampas',
  buttonColor: 'shadyLady',
}
```

gatsby-config is up. It cannot be syntax error. Maybe some issue with babel, or something?

**Comment by gperl27** [comment]
In your gatsby config, try this:

```
{
      resolve: `gatsby-styled-components-dark-mode`,
      options: {
        light: require(`${__dirname}/src/lightTheme.js`).lightTheme,
        dark: require(`${__dirname}/src/darkTheme.js`).darkTheme,
      },
}
```

And in each of the theme files, make sure to `export` the objects.

**Comment by adiusz** [comment]
I tired this too. It's not that.

**Comment by victoroliveirab** [comment]
I just had the same problem. As you opted to use different files, instead of using
```
export default {
  key: value,
};
```
use
```
module.exports = {
  key: value,
}
```

**Comment by victoroliveirab** [comment]
I just had the same problem. As you opted to use different files, instead of using
`darkTheme.js`
```
export default {
  key: value,
};
```
use
`darkTheme.js`
```
module.exports = {
  key: value,
};
```

**Comment by gperl27** [comment]
@victoroliveirab @adiusz can we mark this as closed? It looks like a configuration issue from the consuming app. If not, please share a full implementation of your code so I can investigate the problem.

**Comment by adiusz** [comment]
yep, it works, we can close. thanks for help.

**Comment by catalina-cimpanu** [comment]
Hi! I stumbled into the same problem... I am trying to use this by using a color palette, but I can't figure it out. The way you show in the docs doesn't work, so i tried as recommended in the above comments with module.exports, situation in which my webside loads, except is dead empty :( 
And i did try quite a lot of variants... in the end i gave it up.

**Comment by gperl27** [comment]
> 
> 
> Hi! I stumbled into the same problem... I am trying to use this by using a color palette, but I can't figure it out. The way you show in the docs doesn't work, so i tried as recommended in the above comments with module.exports, situation in which my webside loads, except is dead empty :(
> And i did try quite a lot of variants... in the end i gave it up.

Mind sharing a link to the code?

**Comment by catalina-cimpanu** [comment]
Hi!

I actually ctrl - z everything :(
I will try to reproduce it and I will upload it on github so you can see.
I'll let you know.

Have a good evening :)

**Comment by catalina-cimpanu** [comment]
> > Hi! I stumbled into the same problem... I am trying to use this by using a color palette, but I can't figure it out. The way you show in the docs doesn't work, so i tried as recommended in the above comments with module.exports, situation in which my webside loads, except is dead empty :(
> > And i did try quite a lot of variants... in the end i gave it up.
> 
> Mind sharing a link to the code?

Hi! So, I find it too complicated to reproduce this, and I found a very simple solution for myself, so I'm gonna give it up. The only thing that for sure doesn't work is to do this:
```
const darkTheme = {
  mainColor: "topaz",
  secondaryColor: "ruby",
}

export default darkTheme
```
Here is my attempt on reproducing the errors : [https://github.com/catalina-cimpanu/gatsby-styled-components-dark-mode-attempt](https://github.com/catalina-cimpanu/gatsby-styled-components-dark-mode-attempt)

I think my project was already way too complicated when I tried implementing this. Probably if I would have started from the beginning it would have been different (although to be honest it didn't work for me in this simple project up there, so i don't know what I was doing wrong). 
What I just noticed though and might be the problem for my confusion is that there is a `layout.tsx` file in `.src/components` and a `layout.jsx` in `.src/`. 

Really sorry, but I had to do so much debugging in the last few days that I just couldn't go further in the debugging of this issue.
